src/GarageOnSequence.java

```diff
@@ -0,0 +1,93 @@
+import components.sequence.Sequence2L;
+
+/**
+ * {@code Garage} represented as a sequence of cars with implementation of the
+ * GarageKernel interface.
+ *
+ * @convention |this| <= capacity() and |this| >= 0
+ * @correspondence this = [car1, car2, ..., carN] where N = |this| and 0 <= i < N
+ */
+public class GarageOnSequence extends GarageSecondary {
+    // Representation of {@code this}
+
+    private Sequence2L<String> cars;
+    private int count;
+    private int capacity;
+
+    private void createNewRep() {
+        this.cars = new Sequence2L<>();
+        this.count = 0;
+        this.capacity = 0;
+    }
+    public GarageOnSequence() {
+        this.createNewRep();
+    }
+    public GarageOnSequence(int capacity) {
+        this.createNewRep();
+        this.capacity = capacity;
+    }
+
+    /**
+     * Standard Methods
+     */
+    @Override
+    public final Garage newInstance() {
```

Comment:
See some of your previous projects for how to implement this. newInstance is not meant to make a copy but rather a new object of the same dynamic type.